task13/index.js

```diff
@@ -0,0 +1,92 @@
+const parent = document.getElementById("js-parent"); 
+
+const appendModalForParent = () => {
+    const modalPlace = document.createElement('section');
+    const modalText = document.createElement('p');
+    const closeBtn = document.createElement('input');
+    modalPlace.style.margin = '0 auto';
+    modalPlace.style.textAlign = "center";
+    modalPlace.style.width = '300px';
+    modalPlace.style.padding = '20px';
+    modalPlace.style.backgroundColor = "lightGray";
+    modalPlace.id = "modalPlace";
+    modalText.textContent = "クリックしてデータを取得します。";
+    closeBtn.style.margin = '0 auto';
+    closeBtn.style.display = "block";
+    closeBtn.id = "closeBtn";
+    closeBtn.type = "button";
+    closeBtn.value = "click";
+
+    parent.appendChild(modalPlace).insertBefore(modalText, modalPlace.firstChild).appendChild(closeBtn);
+};
+
+const appendButtonForParent = () => {
+    const button = document.createElement('input');
+    button.id = "button";
+    button.type = "button";
+    button.value = "click";
+    parent.appendChild(button);
+};
+appendButtonForParent();
+
+const modalEvent = document.getElementById("button"); 
+modalEvent.addEventListener('click', (e) => {
+    appendModalForParent();
+    e.target.remove();
+    const closeEvent = document.getElementById("closeBtn");
+    closeEvent.addEventListener('click', () => {
+        document.getElementById('modalPlace').remove();
+        init();
+    });
```

Comment:
Here, it looks like we can separate the functions.
Function and variable names are up to you.😂

IMO
```js
const openBtn = document.getElementById("button");
openBtn.addEventListener("click", (e) => {
  appendModalForParent();
  e.target.remove();
  modalClose();
});

const modalClose = () => {
  const closeBtn = document.getElementById("closeBtn");
  closeBtn.addEventListener("click", () => {
    document.getElementById("modalPlace").remove();
    init();
  });
};
```
[codesandbox](https://codesandbox.io/s/gifted-feynman-icnre?file=/index.js)

Reply:
I was lost in naming!
Thanking for pointing that out!
It is easier to understand if the functions are separated.
Thank you very much.